src/nvim/log.h

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <stdbool.h>
 
+#include "nvim/os/os_defs.h"
```

Comment:
Well since `__func__` is currently broken in C mode on MSVC I think we still need it.

I did open a connect issue for this problem but who knows if it will get  fixed or not. So we will need to define `__func__` ourselves I guess.